config.py

```diff
@@ -118,6 +123,11 @@ class DevelopmentConfig(Config):
     PARTY_PASSWORD = os.getenv('PARTY_PASSWORD', 'secret')
     PARTY_AUTH = (PARTY_USERNAME, PARTY_PASSWORD)
 
+    NOTIFICATION_TEMPLATE_ID = os.getenv('NOTIFICATION_TEMPLATE_ID', "test_notification_template_id")
+
+    RM_NOTIFY_GATEWAY_URL = os.getenv(
+        'RM_NOTIFY_GATEWAY_URL', "http://notifygatewaysvc-dev.apps.devtest.onsclofo.uk/emails/")
```

Comment:
Is there a reason we are pointing this at the notify  gateway in the dev space in cloudfoundry?

Reply:
Not sure I catched the question. Where should point ? This is the same address everywhere in other porjects too.

Reply:
You can keep it like this. Just seems odd when there is an rm-notify-service locally with docker.

Reply:
I think that this should be pointing to localhost, in line with the other URLs in the DevelopmentConfig.  It seems a bit confusing/misleading when we're pointing to a specific dev environment just for this one.

Reply:
@gemmai95  sorry just sene the URL you were mentioning . RM_NOTIFY_GATEWAY_URL . I need to have it Jenkins defo. 
@gavinleeedwards  it s a dev url , does nto need to be localhost. If there is any chance to test this app is just about via that url , not localhost.

Reply:
@atrozzo would expect the default to be `http://localhost:8181` for dev.